lib/widgets/todo_items.dart

```diff
@@ -24,6 +27,8 @@ class _TodoItemsState extends State<TodoItems> {
 
   @override
   Widget build(BuildContext context) {
+    final TodoProvider provider = Provider.of<TodoProvider>(context, listen: true);
```

Comment:
you can remove the setting of value for `listen` here since it is true by default